src/components/networked-hand-controls.js

```diff
@@ -201,17 +203,19 @@ AFRAME.registerComponent('networked-hand-controls', {
       this.el.setObject3D(this.str.mesh, newMesh);
 
       const handMaterial = newMesh.children[1].material;
-      handMaterial.color = new THREE.Color(this.data.handColor);
+      handMaterial.color = new THREE.Color(this.data.color);
+      this.el.sceneEl.systems.renderer.applyColorCorrection(handMaterial.color);
```

Comment:
Why duplicate this code instead of calling `updateHandMeshColor()` ?

Reply:
We already have the handMaterial variable here and we set the color already, calling `updateHandMeshColor()` to get again the mesh, setting the color that didn't change just to convert the color with applyColorCorrection seems the real duplication to me.

Reply:
I defined `this.rendererSystem` is init for better readability.